remi/gui.py

```diff
@@ -496,369 +507,701 @@ class Widget(Tag, EventSource):
+    def css_font_family(self):
+        return self.style.get("font-family", None)
+
     @css_font_family.setter
-    def css_font_family(self, value): self.style['font-family'] = str(value)
+    def css_font_family(self, value):
+        self.style["font-family"] = str(value)
+
     @css_font_family.deleter
-    def css_font_family(self): del self.style['font-family']
+    def css_font_family(self):
+        del self.style["font-family"]
 
     @property
-    @editor_attribute_decorator("Font", '''Font size''', 'css_size', {})
-    def css_font_size(self): return self.style.get('font-size', None)
+    @editor_attribute_decorator("Font", """Font size""", "css_size", {})
+    def css_font_size(self):
+        return self.style.get("font-size", None)
+
     @css_font_size.setter
-    def css_font_size(self, value): self.style['font-size'] = str(value)
+    def css_font_size(self, value):
+        self.style["font-size"] = str(value)
+
     @css_font_size.deleter
-    def css_font_size(self): del self.style['font-size']
+    def css_font_size(self):
+        del self.style["font-size"]
 
     @property
-    @editor_attribute_decorator("Font", '''The line height in pixels''', 'css_size', {})
-    def css_line_height(self): return self.style.get('line-height', None)
+    @editor_attribute_decorator("Font", """The line height in pixels""", "css_size", {})
+    def css_line_height(self):
+        return self.style.get("line-height", None)
+
     @css_line_height.setter
-    def css_line_height(self, value): self.style['line-height'] = str(value)
+    def css_line_height(self, value):
+        self.style["line-height"] = str(value)
+
     @css_line_height.deleter
-    def css_line_height(self): del self.style['line-height']
+    def css_line_height(self):
+        del self.style["line-height"]
 
     @property
-    @editor_attribute_decorator("Font", '''Style''', 'DropDown', {'possible_values': ('normal', 'italic', 'oblique', 'inherit')})
-    def css_font_style(self): return self.style.get('font-style', None)
+    @editor_attribute_decorator("Font", """Style""", "DropDown", {"possible_values": ("normal", "italic", "oblique", "inherit")})
+    def css_font_style(self):
+        return self.style.get("font-style", None)
+
     @css_font_style.setter
-    def css_font_style(self, value): self.style['font-style'] = str(value)
+    def css_font_style(self, value):
+        self.style["font-style"] = str(value)
+
     @css_font_style.deleter
-    def css_font_style(self): del self.style['font-style']
+    def css_font_style(self):
+        del self.style["font-style"]
 
     @property
-    @editor_attribute_decorator("Font", '''Style''', 'DropDown', {'possible_values': ('normal', 'bold', 'bolder', 'lighter', '100', '200', '300', '400', '500', '600', '700', '800', '900', 'inherit')})
-    def css_font_weight(self): return self.style.get('font-weight', None)
+    @editor_attribute_decorator(
```

Comment:
PEP8 is fine but sometimes it makes the code "unreadable". Look here, class properties here are taking lot of lines. Of course there lines are useless, you don't need to look at them, are simple declarations and it is prefereable to get them compact.

Reply:
More than PEP8 this is more of an issue of automatic PEP8 formatting. PEP8 allows for many "correct" formats.
For example, here we could do an intermediate approach as:
```python
    @editor_attribute_decorator("Font", '''Style''', 'DropDown',
                                {
                                    'possible_values':
                                    (
                                        'normal', 'bold', 'bolder', 'lighter',
                                        '100', '200', '300', '400', '500',
                                        '600', '700', '800', '900',
                                        'inherit'
                                    )
                                })
    def css_font_weight(self):
        return self.style.get('font-weight', None)
```

or:
```python
    @editor_attribute_decorator("Font", '''Style''', 'DropDown',
                                {
                                    'possible_values':
                                    ('normal', 'bold', 'bolder', 'lighter',
                                     '100', '200', '300', '400', '500',
                                     '600', '700', '800', '900',
                                     'inherit')
                                })
    def css_font_weight(self):
        return self.style.get('font-weight', None)
```
or:
```python
    @editor_attribute_decorator("Font", '''Style''', 'DropDown',
                                {'possible_values':
                                    ('normal', 'bold', 'bolder', 'lighter',
                                     '100', '200', '300', '400', '500',
                                     '600', '700', '800', '900',
                                     'inherit')})
    def css_font_weight(self):
        return self.style.get('font-weight', None)
```

All these options keep readability (allowing different tastes on it) and also follows PEP8. I, manually, sorted what I think were conceptually similar elements and then autoformatted (via Sublime Text with the Python plugin `Anaconda` hitting Control+Alt+R) the rest of it.

When proposing to move a project into following PEP8, in my experience, a useful approach is to not do all of it in 1 commit and invest time in every meaningful file to be formatted in a way the maintainers keeps track of the code shape.